src/Microsoft.AspNet.Mvc.Razor/Razor/PreCompileViews/RazorPreCompiler.cs

```diff
@@ -18,23 +19,26 @@ public class RazorPreCompiler
         private readonly IServiceProvider _serviceProvider;
         private readonly IFileSystem _fileSystem;
         private readonly IMvcRazorHost _host;
+        private readonly CompilationSettings _compilationSettings;
```

Comment:
This should probably be available in a protected property since it's used in a protected virtual method.